### PR TITLE
Update Critical Articles in the DNS Section

### PR DESCRIPTION
**Pull request by itsalyse**
Updates the following articles marked `Critical`

SRV Records - https://docs.google.com/document/d/1cf8BXg6TSnSwGsDlG012P2_Potq9ta2eAHFo1Pn13FI/edit#heading=h.c2kn3oo3ft7x

Managing A Records - https://docs.google.com/document/d/1Oh5gjjFNYwK2AQQfQ5ou67ihkkAbpiH-JzQ-AdMsEPw/edit

Managing AAAA Records - https://docs.google.com/document/d/1UHoGDBxbrY-5b4SS67DC2W-4p7olu8HYN6iqEgBueQ4/edit#heading=h.lfdicuawme57

Managing CNAME Records - https://docs.google.com/document/d/1NwJpbc-tUW2C0NjmjpCgTsK3ExCB77eGARf9Titj_SY/edit
 
Managing DS Records - https://docs.google.com/document/d/1gEehJ6ZXAb0vaCqCtxqBi-U5VC4nD0aCb4XNJVM03PM/edit

Belongs to https://github.com/dnsimple/dnsimple-business/issues/1900

### CODE REVIEW

File: content/articles/srv-record.md [hunk 1]
```diff
@@ -25,19 +25,32 @@ Here's an example of two SRV records.
...
+## Adding an SRV record
 
-## Add an SRV record
+To add an SRV record in DNSimple:
 
-To add an SRV record in DNSimple, you might fill in the new SRV record form like this:
+1. On your Domain List, click the relevant domain. 
+1. Click the **DNS** badge at the top right to access the [Record Editor](/articles/record-editor/).
+
+<!--- needs screenshot -->
```
Comment (itsalyse) [inline content/articles/srv-record.md:41]: https://drive.google.com/file/d/1-lmjzhqCcbf9-F-9Ur9z8PzAVe73wn95/view?usp=sharing

File: content/articles/manage-a-record.md [hunk 2]
```diff
@@ -26,30 +26,31 @@ The instructions in this article assume you're familiar with the [A record forma
 
 1.  In the record editor, click <label>Add</label> and select <label>A</label> to add a new A record.
 
-    ![screenshot: Showing select menu for choosing to create a new A record](/files/record-a-create-select.png)
+    <!--- needs screenshot -->
```
Comment (itsalyse) [inline content/articles/manage-a-record.md:29]: https://drive.google.com/file/d/1w-escwFAr60YPLBIUpOC8Uqh1ngbzjRQ/view?usp=sharing

File: content/articles/manage-a-record.md [hunk 3]
```diff
@@ -26,30 +26,31 @@ The instructions in this article assume you're familiar with the [A record forma
...
+    - **TTL**: The record time-to-live.
+    - **Respond From Regions**: Configure [Regional Records](/articles/regional-records/) if desired.
+    - **Notes**: Optionally include a [record note](/articles/record-notes/).
 
     Once you're ready, click <label>Add Record</label> to confirm and create the record.
 
 1.  The record is created and visible in the record list.
 
-    ![screenshot: Showing an A record for a domain](/files/record-a-item.png)
+   <!--- needs screenshot -->
```
Comment (itsalyse) [inline content/articles/manage-a-record.md:53]: https://drive.google.com/file/d/180oDQhlZcjtKVrZsj3Io-O0isYBRj-WL/view?usp=sharing

File: content/articles/manage-a-record.md [hunk 4]
```diff
@@ -61,7 +62,7 @@ The instructions in this article assume you're familiar with the [A record forma
 
 1.  In the record editor, search for the record and click on the _pencil_ icon to edit it.
 
-    ![screenshot: Showing button for editing an A record from a domain](/files/record-a-item-edit.png)
+   <!--- needs screenshot -->
```
Comment (itsalyse) [inline content/articles/manage-a-record.md:65]: https://drive.google.com/file/d/1igKQXkicogwOH9MSbtDkRHUPKAw6OHzE/view?usp=sharing

File: content/articles/manage-a-record.md [hunk 5]
```diff
@@ -74,7 +75,11 @@ The instructions in this article assume you're familiar with the [A record forma
 
 1.  In the record editor, search for the record and click on the _trash_ icon to delete it.
 
-    ![screenshot: Showing button for deleting an A record from a domain](/files/record-a-item-delete.png)
+   <!--- needs screenshot -->
```
Comment (itsalyse) [inline content/articles/manage-a-record.md:78]: https://drive.google.com/file/d/1cR_ApMembimX1xPZREBCCN3xWuAYOeQY/view?usp=sharing

File: content/articles/manage-aaaa-record.md [hunk 6]
```diff
@@ -24,28 +24,28 @@ The instructions in this article assume you're familiar with the [AAAA record fo
 <div class="section-steps" markdown="1">
 #### To add an AAAA record
 
-1.  In the record editor, click <label>Add</label> and select <label>AAAA</label> to add a new AAAA record.
+1.  In the record editor, click <label>Add record</label> and select <label>AAAA</label> from the dropdown menu to add a new AAAA record.
 
-    ![screenshot: Showing selection of AAAA for a new record for domain](/files/record-aaaa-create-select.png)
+   <!--- needs screenshot -->
```
Comment (itsalyse) [inline content/articles/manage-aaaa-record.md:29]: https://drive.google.com/file/d/1j9C2wFfHYmyamYNd-UvuESYqlKgTPI9Y/view?usp=sharing

File: content/articles/manage-aaaa-record.md [hunk 7]
```diff
@@ -24,28 +24,28 @@ The instructions in this article assume you're familiar with the [AAAA record fo
...
+    - **Respond From Regions**: Configure [Regional Records](/articles/regional-records/) if desired.
+    - **Notes**: Optionally include a [record note](/articles/record-notes/).
 
-    Once you're ready, click <label>Add Record</label> to confirm and create the record.
+    Click <label>Add Record</label> to confirm and create the record.
 
 1.  The record is created and will be visible in the record list.
 
-    ![screenshot: Showing a newly created AAAA record for a domain](/files/record-aaaa-item.png)
+<!--- needs screenshot -->
```
Comment (itsalyse) [inline content/articles/manage-aaaa-record.md:48]: https://drive.google.com/file/d/1vu_Sv1Ez8bttG0whTEyhxFpxdU_d4Pre/view?usp=sharing

File: content/articles/manage-aaaa-record.md [hunk 8]
```diff
@@ -55,11 +55,11 @@ The instructions in this article assume you're familiar with the [AAAA record fo
 <div class="section-steps" markdown="1">
 #### To update an AAAA record
 
-1.  In the record editor, search for the record and click on the _pencil_ icon to edit it.
+1.  In the record editor, search for the record, and click the _pencil_ icon to edit it.
 
-    ![screenshot: Showing button for editing a AAAA record](/files/record-aaaa-item-edit.png)
+<!--- needs screenshot -->
```
Comment (itsalyse) [inline content/articles/manage-aaaa-record.md:60]: https://drive.google.com/file/d/1cnsEQB3EahOQw4gYoL4P5z0DpDCSXDJy/view?usp=sharing

File: content/articles/manage-aaaa-record.md [hunk 9]
```diff
@@ -68,9 +68,13 @@ The instructions in this article assume you're familiar with the [AAAA record fo
 <div class="section-steps" markdown="1">
 #### To remove an AAAA record
 
-1.  In the record editor, search for the record and click on the _trash_ icon to delete it.
+1.  In the record editor, search for the record, and click the _trash_ icon to delete it.
 
-    ![screenshot: Highlighting delete button for a AAAA record](/files/record-aaaa-item-delete.png)
+<!--- needs screenshot -->
```
Comment (itsalyse) [inline content/articles/manage-aaaa-record.md:73]: https://drive.google.com/file/d/1Hc0mIwPcmKE_UkmBrIFifsy-hzbRNBfL/view?usp=sharing

File: content/articles/manage-ds-record.md [hunk 10]
```diff
@@ -17,23 +17,23 @@ categories:
...
-On the DS records page, click <label>Add DS record</label>.
+<info>
+The form fields shown on the Add DS record page will differ based on whether the TLD of the domain requires them to be set up with the DS-Data interface or the KEY-Data interface. This is automatically populated based on the TLD.
+</info>
 
-![Add DS Record](/files/ds-records-add-ds-record.png)
+On the DS records page, click <label>Add DS record</label>.
 
-The Add DS Record page will load. The form fields shown on the Add DS record page will differ based on whether the TLD of the domain requires them to be set up with the DS-Data interface or the KEY-Data interface.
+<!--- needs screenshot -->
```
Comment (itsalyse) [inline content/articles/manage-ds-record.md:36]: https://drive.google.com/file/d/1eKtv49bu1E1v1mugy5jNU0knfcDRwj3C/view?usp=sharing

File: content/articles/manage-cname-record.md [hunk 11]
```diff
@@ -18,34 +18,33 @@ You can manage [CNAME records](/articles/cname-record) in DNSimple using the [DN
...
+    - **Respond From Regions**: Configure [Regional Records](/articles/regional-records/) if desired.
+    - **Notes**: Optionally include a [record note](/articles/record-notes/).
 
-    Once you're ready, click <label>Add Record</label> to confirm and create the record.
+    Click **Add Record** to confirm and create the record.
 
 1.  The record is created and visible in the record list.
 
-    ![screenshot: Showing a CNAME record](/files/record-cname-item.png)
+<!--- needs screenshot -->
```
Comment (itsalyse) [inline content/articles/manage-cname-record.md:47]: https://drive.google.com/file/d/1_4HwhMPCAPWjIi5zmcrOg-jnx2ytOs3t/view?usp=sharing

File: content/articles/manage-cname-record.md [hunk 12]
```diff
@@ -57,7 +56,7 @@ The instructions in this article assume you're familiar with the [CNAME record f
 
 1.  In the record editor, search for the record and click on the _pencil_ icon to edit it.
 
-    ![screenshot: Showing button to edit a CNAME record](/files/record-cname-item-edit.png)
+<!--- needs screenshot -->
```
Comment (itsalyse) [inline content/articles/manage-cname-record.md:59]: https://drive.google.com/file/d/13TYPtEQkqMEsKeh6vMjCjqDs_D7KRXCB/view?usp=sharing

File: content/articles/manage-cname-record.md [hunk 13]
```diff
@@ -70,7 +69,7 @@ The instructions in this article assume you're familiar with the [CNAME record f
 
 1.  In the record editor, search for the record and click on the _trash_ icon to delete it.
 
-    ![screenshot: Showing button to delete a CNAME record](/files/record-cname-item-delete.png)
+<!--- needs screenshot -->
```
Comment (itsalyse) [inline content/articles/manage-cname-record.md:72]: https://drive.google.com/file/d/1By_TQo-jfQDzVAa_LKOvJE1xO-2qEDZd/view?usp=sharing

File: content/articles/srv-record.md (hunk 1 above)
Comment (itsalyse) [reply content/articles/srv-record.md:41]: it looks like this image is also borking the numbering 🫠

File: content/articles/manage-ds-record.md [hunk 14]
```diff
@@ -7,33 +7,38 @@ categories:
...
+## Listing DS records
 
-![List DS Records](/files/ds-records-list.png)
+On the DS Records page, you will see a list of the DS records currently provisioned for the domain.
 
 ## Adding a DS record
 
-On the DS records page, click <label>Add DS record</label>.
+<info>
+The form fields shown on the Add DS record page will differ based on whether the TLD of the domain requires them to be set up with the DS-Data interface or the KEY-Data interface. This is automatically populated based on the TLD.
```
Comment (itsalyse) [inline content/articles/manage-ds-record.md:36]: I found this little nugget buried in the steps. Felt important so I upgraded it

**Review by Pemacaa — APPROVED** [review]

File: content/articles/manage-ds-record.md (hunk 14 above)
Comment (Pemacaa) [reply content/articles/manage-ds-record.md]: Good call

**Review by Pemacaa — APPROVED** [review]

**Review by stephanoodle — APPROVED** [review]

**Review by dperrymorrow — APPROVED** [review]

**Comment by dperrymorrow** [comment]
have updated / compressed the images

**Pull request merged**